### ISSUE TITLE
FileSystemContentRepositoryTest#testDeleteLinkedResource randomly failing

### ISSUE THREAD

**Issue by myniva** [digitalroastery/weblounge#368]
The following unit test (on Oracle JDK 7?) is randomly failing:

```
ch.entwine.weblounge.contentrepository.fs.FileSystemContentRepositoryTest#testDeleteLinkedResource:430

java.lang.AssertionError:
at org.junit.Assert.fail(Assert.java:91)
at org.junit.Assert.assertTrue(Assert.java:43)
at org.junit.Assert.assertNotNull(Assert.java:524)
at org.junit.Assert.assertNotNull(Assert.java:535)
at ch.entwine.weblounge.contentrepository.fs.FileSystemContentRepositoryTest.testDeleteLinkedResource(FileSystemContentRepositoryTest.java:430)
```

See https://travis-ci.org/entwinemedia/weblounge/jobs/47613723 for example

**Comment by myniva** [comment]
Test failed again in https://travis-ci.org/entwinemedia/weblounge/jobs/48016180

**Comment by myniva** [comment]
Fixed by #382
